src/Template/Layout/js/app/directives/richeditor.js

```diff
@@ -12,18 +11,38 @@ export default {
              *
              * @param {Object} element DOM object
              */
-            inserted (el) {
-                import(/* webpackChunkName: "richtext-editor-input" */'app/components/richtext-editor-input')
-                    .then(module => module.default)
-                    .then((component) => {
-                        const Constructor = Vue.extend(component);
-                        const vm = new Constructor({
-                            propsData: {
-                                el,
-                            }
-                        });
-                        vm.$mount();
-                    });
+            async inserted(el) {
+                const { ClassicEditor, ...plugins } = await import(/* webpackChunkName: "ckeditor" */'app/lib/ckeditor');
+
+                ClassicEditor.create(el, {
+                    plugins: Object.values(plugins),
+                    toolbar: {
```

Comment:
In BE3 avevamo diverse configurazioni dell'editor. Non so se è già il caso di spostare questa conf altrove, in una costante di default conf, e qui specificare quale utilizzare.